Bottom: ef75c87ba4586a879f503ee1b10a162b33a420e3
Top:    e32560bf251105655d134b5c41a9b8fdd9801d15
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:37:18 -0300

Refresh of manual-document-mt-safety-filesys.patch

---

diff --git a/manual/filesys.texi b/manual/filesys.texi
index 2244025..47176cc 100644
--- a/manual/filesys.texi
+++ b/manual/filesys.texi
@@ -1034,7 +1034,7 @@ transparently replaces the old implementation.
 @c open_dir_stream calls malloc, realloc, readdir64, free, closedir,
 @c  then openat64_not_cancel_3 and fdopendir or opendir, then dirfd.
 @c process_entry may cal realloc, fxstatat/lxstat/xstat, ftw_dir, and
-@c  find_object (tsearch) and add_object (tfind).  
+@c  find_object (tsearch) and add_object (tfind).
 @c Since each invocation of *ftw uses its own private search tree, none
 @c  of the search tree concurrency issues apply.
 The @code{nftw} function works like the @code{ftw} functions.  They call
